shell/common/shell.cc

```diff
@@ -789,13 +789,17 @@ void Shell::OnAnimatorDraw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline) {
       [& waiting_for_first_frame = waiting_for_first_frame_,
        &waiting_for_first_frame_condition = waiting_for_first_frame_condition_,
        rasterizer = rasterizer_->GetWeakPtr(),
+       &first_frame_callback = first_frame_callback_,
```

Comment:
rather than keep track and firing back here again, can we just leverage rasterizer->SetNextFrameCallback and set that when !first_frame_rasterized_ or some such?